src/lib/support/IntrusiveList.h

```diff
@@ -29,7 +29,7 @@ class IntrusiveListNodeBase
 {
 public:
     IntrusiveListNodeBase() : mPrev(nullptr), mNext(nullptr) {}
-    ~IntrusiveListNodeBase() { VerifyOrDie(!IsInList()); }
+    ~IntrusiveListNodeBase() {}
```

Comment:
The verify or die here was on purpose I believe to enforce freeing. Removing it indicates potential problems in the code.